### ISSUE TITLE
Provide image as zip

### ISSUE THREAD

**Issue by probonopd** [distr1/distri#43]
https://repo.distr1.org/distri/jackherer/img/ has filesystem images; however they are in formats that [balenaEtcher](https://www.balena.io/etcher/) (the cross-platform quasi-standard tool to write images to USB devices) cannot understand (without the user having to unpack them first). Would it possible to have a `.img.zip` that balenaEtcher can work on?

**Comment by stapelberg** [comment]
This is the first time I hear of balenaEtcher :)

While it would of course be possible to provide a .zip file, there are downsides (more disk space and bandwidth necessary for mirrors).

Can you explain why you can not just manually extract one of the existing archive formats? balenaEtcher should take a raw disk image, right?

**Comment by probonopd** [comment]
Yes, but it means that I have to run gunzip, and this means it takes a _lot_ of time and 8.5 GB of temp space on the hard disk. Inconvenient if you are running your OS e.g., from a Live ISO where you have maybe 1-2 GB of temp space but not 8.5 GB...

**Comment by mohe2015** [comment]
You could pipe the output, something like (NOT TESTED)
`wget -O - https://repo.distr1.org/distri/jackherer/img/distri-qemu-serial.img.gz | gunzip | dd bs=512k of=/dev/null`

**Comment by probonopd** [comment]
Great trick @mohe2015, still I'd like to see this work in Etcher...

**Comment by mohe2015** [comment]
I'm pretty sure balenaEtcher supports img.zg images (https://github.com/balena-io/etcher/issues/325) which are provided by https://repo.distr1.org/distri/jackherer/img/. I also just ran it and it tells me that it would burn the file without the .gz extension so I think thats right.
